### PR TITLE
VSIX-54: Move build from TeamCity to GitHub Actions

### LINKED ISSUE #26: Move build from TeamCity to GitHub Actions (VSIX-54)
With artifact publishing for tagged (release) builds.

[Jira issue](https://lombiq.atlassian.net/browse/VSIX-54)

### PR DESCRIPTION
**Pull request by Piedone**
VSIX-54
Fixes #26.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
+
+# Mostly buit on https://github.com/timheuer/JwtTools/blob/main/.github/workflows/build.yaml.
```
Comment (BenedekFarkas) [inline .github/workflows/build.yml]:
```suggestion
# Mostly built on https://github.com/timheuer/JwtTools/blob/main/.github/workflows/build.yaml.
```

File: Marketplace/Overview.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Visual Studio extension with many features and templates frequently used by�[Lombiq](https://lombiq.com/)�developers. Contains�[Orchard CMS](https://www.orchardcore.net/)-related (including Orchard Core and Orchard 1.x) as well as generic goodies. For Orchard developers and for other .NET developers alike!
```
Comment (BenedekFarkas) [inline Marketplace/Overview.md]: Seems like we've got a couple of gremlins (there's more below).

File: .github/actions/build-vsix/action.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+name: Build VSIX
+description: Builds the Visual Studio extension package for the solution.
+
+inputs:
+  version-number:
+    description: >
+      If provided, will be used directly as the version of the VSIX. Otherwise, the version will be automatically
+      incremented.
+    required: false
+    default: ''
```
Comment (BenedekFarkas) [inline .github/actions/build-vsix/action.yml]: The action/workflow files in the PR have a mixed usage of `'` and `"`, let's stick to `"`.

**Review by BenedekFarkas — CHANGES_REQUESTED** [review]

File: Marketplace/Overview.md (hunk 2 above)
Comment (Piedone) [reply Marketplace/Overview.md]: WTF :(.

**Pull request merged**